meetings/2021/2021-01-05.md

```diff
@@ -54,11 +54,10 @@
 
 * Improving Diversity and Inclusion at the OpenJSF
 [#699](https://github.com/openjs-foundation/cross-project-council/issues/699)
-  * Link shared by Robin: https://openjsf.org/blog/2020/12/14/training-course-on-diversity-in-open-source
+  * Link shared by Robin: https://openjsf.org/blog/training-course-on-diversity-in-open-source
   * Link shared by Robin: https://inclusivenaming.org
   * Prompt from Tierney: do we have materials to give to member companies which could also be used by non-member companies to show how leadership provides value in open source projects; specifically around URM being able to show their work/effort/participation
   * Robin will pull data from LF Insights
-  * Brian shared: https://insights.lfx.linuxfoundation.org/projects/ojsf-f
```

Comment:
Can someone verify that the OJSF project was removed from the LF Insights portal? I couldn't find it. cc @bensternthal

Reply:
it definitely doesn't seem to be on https://insights.lfx.linuxfoundation.org